packages/story-editor/src/components/header/buttons/buttonWithChecklistWarning.js

```diff
@@ -114,22 +122,33 @@ function ButtonWithChecklistWarning({
     toolTip = TOOLTIP_TEXT_REVIEW;
   }
 
+  const publishText = useMemo(
+    () =>
+      hasFutureDate && status !== 'private'
+        ? __('Schedule', 'web-stories')
+        : __('Publish', 'web-stories'),
+    [hasFutureDate, status]
+  );
+
+  const text = canPublish
+    ? publishText
+    : __('Submit for review', 'web-stories');
   return (
-    <Tooltip title={toolTip} hasTail>
+    <Tooltip title={toolTip} popupZIndexOverride={11} hasTail>
       <InnerButton
         text={text}
-        {...buttonProps}
+        disabled={disabled || isSaving || isUploading}
```

Comment:
pulled `disabled` out from `buttonProps` so that it can be one of these 3 values that would set the button to be disabled.